image_analysis_configs/undulatorrad2cam_may2_settings.ini

```diff
@@ -7,6 +7,8 @@ calibration_0th_order_pixel = 289.4770
 minimum_wavelength_analysis = 150.0
 optimization_central_wavelength = 420.0
 optimization_bandwidth_wavelength = 10.0
+spec_correction_bool = False
+spec_correction_files = []
```

Comment:
While we aren't using the python analysis for Rad2 at the moment, it will complain if the .ini file variable names don't match what is in `__init__`